android-components/components/concept/engine/src/main/java/mozilla/components/concept/engine/EngineSession.kt

```diff
@@ -474,14 +474,19 @@ abstract class EngineSession(
              */
             MOZILLA_SOCIAL(1 shl 8),
 
+            /**
+             * Blocks email trackers.
+             */
+            EMAIL(1 shl 9),
```

Comment:
Do you know where in Gecko we are adding this new tracking category?  :) 

Normally all the values that we are exposing here are coming from [ContentBlocking](https://searchfox.org/mozilla-central/source/mobile/android/geckoview/src/main/java/org/mozilla/geckoview/ContentBlocking.java#1195-1230).

We also have [tests](https://github.com/mozilla-mobile/firefox-android/blob/56a6e6dbb55bd5e0c8966a90cabd64b481d95702/android-components/components/browser/engine-gecko/src/test/java/mozilla/components/browser/engine/gecko/GeckoEngineSessionTest.kt#L1720) to make sure those values are aligned with what we are getting from GeckoView.

It's a bit odd that we are adding on AC first, without having a similar value on GeckoView as this tracking categories will [eventually go to GV](https://searchfox.org/mozilla-central/source/mobile/android/geckoview/src/main/java/org/mozilla/geckoview/ContentBlocking.java#458-469)

Reply:
circling back now that the GeckoView portion of this new tracking category has been added.

This patch will now make more sense.  I've also updated the tests for the `ContentBlocking` values.